data/src/languages.json

```diff
@@ -1385,24 +1964,33 @@
         "skip_spaces_word": false,
         "skip_spaces_pron": false,
         "dialect": {
-            "hanoi": "Hà Nội",
-            "hcmc": "Hồ Chí Minh City",
-            "hue": "Huế",
-            "tc": "Vinh, Thanh Chương",
-            "ht": "Hà Tĩnh"
+            "hanoi": "H\u00e0 N\u1ed9i",
```

Comment:
Strings not output correctly.

Reply:
yeah, I forgot to include `ensure_ascii=False` in json.dumps, but I ran it with that param, and that fixed it.